### PR TITLE
yt_idv updates from Aidan

### PR DESCRIPTION
**Pull request by chrishavlin**
This is a submission on behalf of Aidan Sochowski! Aidan wrote the post, but I'll handle any editing required. 

This post does include some gifs. They autoplayed successfully in a local hugo server, so hopefully they render here OK too!

### CODE REVIEW

**Comment by brittonsmith** [comment]
This looks great. My only suggestion would be to add links to any PRs associated with the work.

**Review by brittonsmith — APPROVED** [review]

**Comment by chrishavlin** [comment]
> This looks great. My only suggestion would be to add links to any PRs associated with the work. 

Great suggestion, I'll add those links.

File: content/posts/yt_idv_spin.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+A brief overview of my contributions to yt-idv this past semester.
+
+<!--more-->
+
+# My semester working on yt-idv 
+This past semester I worked on adding a few new features to [yt-idv](https://github.com/yt-project/yt_idv) through NCSA's [Student Pushing INnovation (SPIN)](http://spin.ncsa.illinois.edu/) internship program. Since I just wrapped up my semester, here's a quick summary of the largest changes I've been working on.
+
+## Isocontours
+By far the largest feature I was working on this semester was adding isocontours to yt-idv. Isocontours serve as a method to view any level set of your data by simply setting some value and tolerance
```
Comment (neutrinoceros) [inline content/posts/yt_idv_spin.md]:
```suggestion
By far the largest feature I was working on this semester was adding isocontours to yt-idv. Isocontours serve as a method to view any level set of your data by simply setting some value and tolerance.
```

File: content/posts/yt_idv_spin.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+![](/img/yt_idv_spin/blogpost1.gif) 
+
+*A simple single layer isocontour: this gif shows the process of adding a single isocontour layer at a density value of 10^-24, using one of yt's sample datasets, IsolatedGalaxy.*
+
+![](/img/yt_idv_spin/blogpost2.gif) 
+
+*Creating isocontours with multiple layers: this gif shows the process of adding and deleting multiple isocontour layers, demonstrated on the ActiveParticleTwoSphere sample dataset.*
+
+## Spherical Rendering
+The other main feature I worked on this semester was spherical rendering in yt-idv. Many datasets are natively stored as (r, theta, phi) spherical coordinates rather than as (x, y, z) cartesian coordinates, so it would make sense for yt-idv to support these datasets, letting the user avoid the hassle of converting these datasets to utilize a cartesian coordinate system instead.
```
Comment (neutrinoceros) [inline content/posts/yt_idv_spin.md]: YES ! I'm really excited about this feature (albeit incomplete right now) !

**Review by neutrinoceros — APPROVED** [review]
Only found one minuscule typo. Otherwise this looks great, bravo Aidan !

File: content/posts/yt_idv_spin.md (hunk 2 above)
Comment (chrishavlin) [reply content/posts/yt_idv_spin.md]: YES! Me too! I think it's not too far off. Aidan's initial work definitely moved the timeline up **significantly**

**Comment by chrishavlin** [comment]
PR links now added!

**Pull request merged**